File: Easy/Python/2574. Left and Right Sum Differences.py
class Solution(object):
    def leftRigthDifference(self, nums):
        left = 0
        right = 0

        for n in nums:
            right += n
        
        for i in range(len(nums)):
            left += nums[i]
            right -= nums[i]
            nums[i] = abs((left - nums[i]) - right)

        return nums

File: Easy/Java/2574. Left and Right Sum Differences.java
class Solution {
    public int[] leftRigthDifference(int[] nums) {
        int left = 0;
        int right = 0;
        
        for(int n : nums){
            right += n;
        }

        for(int i = 0; i < nums.length; i++){
            left += nums[i];
            right -= nums[i];
            nums[i] = Math.abs((left - nums[i]) - right);
        }

        return nums;
    }
}
